### PR TITLE
#feat[#I1.3]: Testing registration with pharmacy

### PR DESCRIPTION
**Pull request by mladenkak**

### CODE REVIEW

File: PatientWebApplication/HealthClinic.DAL/Service/RegistrationInPharmacyService.cs [hunk 1]
```diff
@@ -25,17 +30,30 @@ public List<RegistrationInPharmacy> GetAll()
...
-            return RegistrationInPharmacyRepository.getApiKey(pharmacyId);
+            return IRegistrationRepository.GetAll();
         }
-        public RegistrationInPharmacy getPharmacyId(String apiKey)
-        {
-            return RegistrationInPharmacyRepository.getPharmacyId(apiKey);
+
+        public RegistrationInPharmacy getPharmacyApiKey(String apiKey)
+        {
+            List<RegistrationInPharmacy> registrationInPharmacies = IRegistrationRepository.GetAll();
```
Comment (Nemanja0505) [inline PatientWebApplication/HealthClinic.DAL/Service/RegistrationInPharmacyService.cs:51]: Unnecessary instance of the List registrationInPharmacies. Do the same as in the 38 line of code.

File: PatientWebApplication/HealthClinic.DAL/Repository/RegistrationInPharmacyRepository.cs [hunk 2]
```diff
@@ -24,14 +24,14 @@ public RegistrationInPharmacy Create(RegistrationInPharmacy registration)
...
-            return dbContext.Registrations.SingleOrDefault(registration => registration.pharmacyId == pharmacyId);
-        }
-        public RegistrationInPharmacy getPharmacyId(String apiKey)
+        }
+       
+        public RegistrationInPharmacy getPharmacyApiKey(String apiKey)
         {
-            return dbContext.Registrations.SingleOrDefault(registration => registration.apiKey == apiKey);
-        }
+            Console.WriteLine(dbContext.Registrations.ToList().SingleOrDefault(registration => registration.apiKey == apiKey));
```
Comment (Nemanja0505) [inline PatientWebApplication/HealthClinic.DAL/Repository/RegistrationInPharmacyRepository.cs:31]: Delete unnecessary printouts on the console.

**Review by Nemanja0505 — CHANGES_REQUESTED** [review]
It is necessary to fix the indicated parts of the code.

**Review by Nemanja0505 — APPROVED** [review]
All clean code principles are met, therefore PR is approved.

**Review by trki293 — APPROVED** [review]
PR is approved, all principles of Clean code are followed.

**Pull request merged**